frontend/Views/ForumFeed/constants.js

```diff
@@ -3,10 +3,12 @@ export const STOP_FETCHING_DISCUSSIONS = 'stop_fetching_discussions';
 export const FETCHING_DISCUSSIONS_SUCCESS = 'fetching_discussions_success';
 export const FETCHING_DISCUSSIONS_FAILURE = 'fetching_discussions_failure';
 
+
 export const START_FETCHING_PINNED_DISCUSSIONS = 'start_fetching_pinned_discussions';
 export const STOP_FETCHING_PINNED_DISCUSSIONS = 'stop_fetching_pinned_discussions';
 export const FETCHING_PINNED_DISCUSSIONS_SUCCESS = 'fetching_pinned_discussions_success';
 export const FETCHING_PINNED_DISCUSSIONS_FAILURE = 'fetching_pinned_discussions_failure';
 
 export const UPDATE_SORTING_METHOD = 'update_sorting_method';
 export const INVALID_FORUM = 'invalid_forum';
+export const SEARCH_FILTERING_SYNC = "search";
```

Comment:
nit: the actual value of the string here should be exactly the same as the variable name but lowercase. Thanks for making the change!

Reply:
OK. Changed the name to search_filtering_sync in all files